node/resolvers/Routes/index.ts

```diff
@@ -161,6 +161,7 @@ export const Routes = {
     let tradeName = null
     let stateRegistration = null
     let user = null
+    let isCorporate = false
```

Comment:
It is better to use immutable variables if the value never changes. What do you think to declare instead of the `if`?
```Typescript
const isCorporate = organizaitons ? true : false
```